<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StandardDialogDemoClass</class>
 <widget class="QWidget" name="StandardDialogDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StandardDialogDemo</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>文件对话框</string>
      </attribute>
      <widget class="QPushButton" name="btnOpenFile">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>选择打开一个文件</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnOpenFiles">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>20</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>选择打开多个文件</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSaveFile">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>选择保存一个文件</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSelDir">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>选择一个已有的目录</string>
       </property>
      </widget>
      <widget class="QPushButton" name="OpenFileUrl">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>191</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>选择打开一个远程网络的文件</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>颜色对话框</string>
      </attribute>
      <widget class="QPushButton" name="btnPickColor">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>选择颜色</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="ptShowColor">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>131</width>
         <height>101</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Aharoni</family>
         <pointsize>28</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="plainText">
        <string>123456</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>字体对话框</string>
      </attribute>
      <widget class="QPushButton" name="btnPickFont">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>选择字体</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="ptShowFont">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>131</width>
         <height>101</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Aharoni</family>
         <pointsize>28</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="plainText">
        <string>123456</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>输入对话框</string>
      </attribute>
      <widget class="QPushButton" name="btnInputLine">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>输入单行文本</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnInputInt">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>输入整数</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnInputFloat">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>50</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>输入浮点数</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnGetItem">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>下拉框方式选择输入</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnInputLines">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>20</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>输入多行文本</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="ptShowInput">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>271</width>
         <height>111</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Aharoni</family>
         <pointsize>28</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>消息框</string>
      </attribute>
      <widget class="QPushButton" name="btnInfoMsg">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>信息提示对话框</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnQuestMsg">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>询问对话框</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnWarningMsg">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>警告提示对话框</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnCriticalMsg">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>错误提示对话框</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnAboutMsg">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>关于对话框</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnAboutQtMsg">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>关于QT对话框</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="StandardDialogDemo.qrc"/>
 </resources>
 <connections/>
</ui>
